proto/zetachain/zetacore/crosschain/tx.proto

```diff
@@ -184,6 +184,9 @@ message MsgVoteInbound {
 
   // define if a smart contract call should be made with asset
   bool is_cross_chain_call = 19;
+
+  // it carries the error message that caused the inbound to fail
+  string error_message = 20;
```

Comment:
Same, to replace with a status enum

Reply:
replaced in commit https://github.com/zeta-chain/node/pull/3358/commits/3b44dc6cf18412c681ca31b4917fdcb8fccf720f